sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -385,6 +386,28 @@ public <T> Mono<List<State<T>>> getBulkState(GetBulkStateRequest request, TypeRe
   }
 
 
+  @Override
+  public Mono<DaprMetadata> getMetadata() {
+    
+    String[] pathSegments = new String[] { DaprHttp.API_VERSION, "metadata"};
+
+    return Mono.deferContextual(context ->  this.client.invokeApi(DaprHttp.HttpMethods.GET.name(), pathSegments,
+            null, "", null, null))
+            .flatMap(r -> {
+              try {
+                return Mono.just(buildMetadata(r));
+              } catch (Exception ex) {
+                return DaprException.wrapMono(ex);
+              }
+            });
+  }
+
+  private DaprMetadata buildMetadata(
+      DaprHttp.Response response) throws IOException {
+
```

Comment:
Same here, `new TypeRef<DaprMetadata>() {}` could be extracted as a constant, so we don't have to create an new `TypeRef` object for every invocation.

Reply:
Same here.. the comment doesn't show they line that you are referring too.